es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
-**1. Machine Learning tips and tricks cheatsheet**
+**1. Consejos y trucos sobre Aprendizaje Automático**
```

Comment:
Would it be correct to say "Hoja de referencia de consejos y trucos sobre Aprendizaje Automático" in order to normalize it with the other files, with cheatsheet = hoja de referencia?